css/theme.css

```diff
@@ -42,6 +42,10 @@ body {
   border-radius: 6px;
 }
 
+body.use-native-titlebar {
+  border-radius: 0;
+}
```

Comment:
@paulrouget Can you write a comment explaining the story about the border-radius here ? I have no idea why it has being there in first place or why is it that we don't want it if native titlebar is used.